tests/cities.test.js

```diff
@@ -0,0 +1,97 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const Cities = require('../src/models/cities');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const Cities = require("../src/models/cities");
```